ctms/schemas/contact.py

```diff
@@ -139,6 +155,20 @@ class CTMSSingleResponse(CTMSResponse):
     )
 
 
+class CTMSBulkResponse(BaseModel):
+    """
+    Response for GET /bulk_ctms/
+
+    """
+
+    start: datetime
+    end: datetime
+    limit: int
+    after: Optional[str] = None
+    next: Optional[Union[AnyUrl, str]] = None
+    items: List[Optional[CTMSOptionalResponse]]
```

Comment:
Should this be ``List[CTMSOptionalResponse]``? I'm not sure what the ``Optional`` is for.

Reply:
The intent of this optional was for a blank list; as a List[CTMSOptionalResponse] I think still had a requirement on their being some inner value. Can modify to test and confirm/deny